activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNActivitySignaledEvent.java

```diff
@@ -0,0 +1,5 @@
+package org.activiti.api.process.model.events;
+
+public interface BPMNActivitySignaledEvent extends BPMNActivityEvent {
```

Comment:
@CTI777 then the same change should be applied here.

Reply:
@CTI777 here you should not extend BPMNActivityEvent, but signal should have his own class (maybe `BPMNSignal`)